rasa/nlu/featurizers/dense_featurizer/convert_featurizer.py

```diff
@@ -87,10 +87,12 @@ def __init__(self, component_config: Optional[Dict[Text, Any]] = None) -> None:
         self.module = train_utils.load_tf_hub_model(self.model_url)
 
         self.tokenize_signature = self._get_signature("tokenize", self.module)
-        self.sequence_encoding_signature = self._get_signature(
+        self.sequence_encoding_signature: Any = self._get_signature(
             "encode_sequence", self.module
         )
-        self.sentence_encoding_signature = self._get_signature("default", self.module)
+        self.sentence_encoding_signature: Any = self._get_signature(
```

Comment:
@ancalita Just curious: How is `Any` a helpful annotation type? Are we just adding it so that mypy does not complain in these?

Reply:
@dakshvar22 I would have added a more informative type but I couldn't figure out what a signature type is 🤔  Any suggestions are welcome, I think it's something from TF?

Reply:
Let me take a quick look at the code.

Reply:
Okay, each of these come out to be of type `<class 'tensorflow.python.eager.wrap_function.WrappedFunction'>`. Not sure what type annotation should be used for them. Haven't checked myself but is `WrappedFunction` a usual python construct?

Reply:
Seems to be a TF specific class - I can add the import and see if it something breaks in the CI - otherwise revert to `Any`?

Reply:
Alright 👍 Let's add `Any` to `self.tokenize_signature` as well just to be consistent.